src/webview/html/messageAsHtml.js

```diff
@@ -134,6 +143,14 @@ $!${divOpenHtml}
   <div class="static-timestamp">${messageTime}</div>
 </div>
 `;
+  // TODO: i18n
```

Comment:
I think this line can be deleted :slightly_smiling_face:

Reply:
Done :sweat_smile: